packages/terra-clinical-embedded-content-consumer/src/EmbeddedContentConsumer.scss

```diff
@@ -0,0 +1,17 @@
+@import '~terra-mixins';
```

Comment:
With the conversion to CSS Modules, we have included a webpack plugin that handles bi-directional styling. `@import '~terra-mixins';` can be removed here and the terra-mixing package dependencies can also be removed.